### ISSUE TITLE
Support for tags above Examples

### ISSUE THREAD

**Issue by josepj** [trivago/cucable-plugin#84]
**Problem:**
I have a scenario outline with tags above Examples as below:
```
@featureTag
Feature: This is the feature name

@scenarioTag
Scenario Outline: Scenario with an amount of <amount>
        Given I am on the start page
        And I add <amount> items
        And I navigate to the shopping basket
        Then I see <amount> items

        @exampleTag01
        Examples:
            | amount |
            | 12     |
            | 85     |

        @exampleTag02
        Examples:
            | amount |
            | 99     |
```
I see that on running the above scenario outline, the individual feature files generated by Cucable include the tags `@featureTag` and `@scenarioTag`, but do not include the tags `@exampleTag01` and `@exampleTag02`. No tests are executed when `--tags @exampleTag01` or `--tags @exampleTag02` are used. When using `--tags @scenarioTag`, only the first `Examples:` table is executed.

It would be nice to have support for Example tags.

**Environment:**
cucumber-java v4.2.0
Cucable v1.2.0
maven-surefire-plugin v3.0.0-M1
maven-failsafe-plugin v3.0.0-M1

**Comment by laxersaz** [comment]
Hi @josepj 
honestly, I didn't know example tags were possible. So I will implement this for sure for the next release. Thanks for pointing this out!

**Comment by laxersaz** [comment]
This is currently in development:

- Implement tag inheritance (feature -> scenario -> examples)
- Consider example tags when passing `includeScenarioTags` and `excludeScenarioTags`

**Comment by laxersaz** [comment]
Hello @josepj ,

this will take some more time as it has to be very carefully implemented. But rest assured I will try to finish this up next week.

Have a good weekend,
Benjamin

**Comment by josepj** [comment]
Thanks @laxersaz! Appreciate it.

**Comment by laxersaz** [comment]
Fixed with 1.3.0

**Comment by josepj** [comment]
@laxersaz, just ran a quick test and the tags work perfectly! Thank you!
